pkg/types.go

```diff
@@ -58,8 +60,10 @@ type WorkflowTemplate struct {
 	Name       string
 	Manifest   string
 	Version    int32
-	IsLatest   bool `db:"is_latest"`
+	IsLatest   bool
 	IsArchived bool `db:"is_archived"`
+	LatestArgo *wfv1.WorkflowTemplate
```

Comment:
Can we use a better field name? Like `ArgoWorkflowTemplate`.

Reply:
On it.

Reply:
Done.